.gitignore

```diff
@@ -30,3 +30,6 @@ test/build
 
 # Misc mac crap
 .DS_Store
+
+# symlinked docker run script
+/run
```

Comment:
Is this something on your own box, or does it get generated somehow?

Reply:
This is the file `test/docker_make_test.sh` which is mapped as a volume in `docker-compose.yml` to `/pgtap/run` inside the container. And because the main directory is mapped as `/pgtap` as well, the file is actually created on the host.

The mapping is just there to be able to run `docker-compose exec test run` instead of `docker-compose exec test test/docker_make_test.sh`.

Reply:
I suggest copying it to some other location; no need to pollute the repo on every run. Does it need to go there? If so, a better more temporary-looking name might be useful.

Reply:
Agreed. The main point why I did it this way, was because I didn't want to put the script in the main directory itself - because it's only needed for running the tests. Having it still show up there, kind of defeats that... ;)

I changed it a bit, I think the way is better now.